anime-list-master.xml

```diff
@@ -43948,6 +43957,9 @@
   </anime>
   <anime anidbid="15964" tvdbid="402117" defaulttvdbseason="1" episodeoffset="" tmdbid="" imdbid="">
     <name>Kawaii dake ja Nai Shikimori-san</name>
+    <mapping-list>
+      <mapping anidbseason="0" tvdbseason="0">;1-4;2-0;3-0;4-5;5-0;</mapping>
```

Comment:
While there's no conflict for the other episodes, these unmappings don't feel as out of place given the "hole" on the TVDB numbering (why on Earth their specials start at 04?! Crazy people) and the show being recent enough for either release new OVAs or TVDB mods freaking out on ordering as they often do.

Reply:
OK, I'll keep this mapping then.